src/settings.rs

```diff
@@ -19,13 +19,21 @@ pub struct Window {
 
     /// Whether the window should be resizable or not.
     pub resizable: bool,
+
+    /// Whether the window should have a border, a title bar, etc.
+    pub decorations: bool,
+
+    /// Platform specific Setting.
+    pub platform_specific: iced_winit::settings::PlatformSpecific,
```

Comment:
The idea is to simply not expose this for now and make advanced users use `iced_winit` directly instead.

Reply:
Make sense!
I've removed extended settings from iced's setting.